templates/account/login.html

```diff
@@ -20,12 +20,12 @@ <h3 class="welcome">Welcome Back!</h3>
             <form method="POST" action="{% url 'account_login' %}">
                 {% csrf_token %}
                 <div class="form-group">
-                    <label for="id_login">Username</label><br>
+                    <!-- <label for="id_login">Username</label><br> -->
                     {{ form.login }}<br><br>
                     {% if redirect_field_value %}
                     <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
                     {% endif %}
-                    <label for="id_password"> Password</label><br>
+                    <!-- <label for="id_password"> Password</label><br> -->
                     {{ form.password }}<br>
                     <a class="forgotpassword" href="{% url 'account_reset_password' %}">{% trans "Forgot Password?" %}</a>
```

Comment:
@maneprajakta Check this. In a previous commit @Abg4real had changed this, so I had reset it back. So my guess is your branch is not up-to-date with remote

Reply:
Ok :+1: @Praful932